src/Schema/Events/Click.ts

```diff
@@ -38,6 +38,32 @@ export interface ClickedArtistGroup extends ClickedEntityGroup {
   action: ActionType.clickedArtistGroup
 }
 
+/**
+ * A user clicks a grouping of artist series on iOS. If the artist series was boosted by the curation team, then featured will be set to true.
+ *
+ *  This schema describes events sent to Segment from [[clickedEntityGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedArtistSeriesGroup",
+ *    context_module: "moreSeriesByThisArtist",
+ *    context_page_owner_type: "artistSeries",
+ *    context_page_owner_id: "5359794d1a1e86c3740001f7",
+ *    context_page_owner_slug: "alex-katz-departure",
+ *    destination_page_owner_type: "artistSeries",
+ *    destination_page_owner_id: "5359794d1a1e86c3740001f7",
+ *    destination_page_owner_slug: "alex-katz-black-dress",
+ *    featured: true,
+ *    horizontal_slide_position: 1,
+ *    type: "thumbnail"
+ *  }
+ * ```
+ */
+export interface ClickedArtistSeriesGroup extends ClickedEntityGroup {
+  action: ActionType.clickedArtistSeriesGroup
```

Comment:
Because this event always has destination `OwnerType.artistSeries`, we can include here to enforce in schema.